src/Models/Refund.php

```diff
@@ -0,0 +1,237 @@
+<?php namespace Tipoff\Refunds\Models;
+
+use Carbon\Carbon;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Support\Str;
+use Stripe\Stripe;
+use Tipoff\Refunds\Notifications\RefundConfirmation;
+use Tipoff\Support\Models\BaseModel;
+
+class Refund extends BaseModel
+{
+    use HasFactory;
+
+    const METHOD_STRIPE = 'Stripe';
+    const METHOD_VOUCHER = 'Voucher';
```

Comment:
This is a candidate for creation of an Enum class for `PaymentMethod`

Reply:
I would call it good practice, not a rule (to accelerate). Most important to use constraints.

I would suggest making it rules on the next steps.

Reply:
Agreed.